packages/loaders/src/elements/Inline.tsx

```diff
@@ -27,7 +27,7 @@ export interface IInlineProps extends React.HTMLAttributes<SVGSVGElement> {
 }
 
 /**
- * All other props are spread onto the root `<svg>` element
+ * @extends HTMLAttributes<SVGSVGElement>
```

Comment:
I think `SVGAttributes<SVGSVGElement>` is preferable, and the `interface` should be modified accordingly.